src/stats.ml

```diff
@@ -45,3 +45,117 @@ let dump () =
 let enable () =
   enabled := true;
   at_exit dump
+
+module Catapult = struct
```

Comment:
Does it make sense to have a `dune.catapult` library perhaps? This seems to be a fairly standalone feature.

Reply:
Good idea! It's also a good occasion to separate this between a global reference (which would live somewhere, maybe in `Stats`?) and the rest of the library code.